docker/docker.go

```diff
@@ -292,19 +292,28 @@ func WithEnv(key, value string) ConfigOption {
 }
 
 func WithVolume(name, containerPath, hostOS string) ConfigOption {
-	return withVolume(mount.TypeVolume, name, containerPath, hostOS)
+	return withVolume(mount.TypeVolume, name, containerPath, false, hostOS)
+}
+
+func WithROVolume(name, containerPath, hostOS string) ConfigOption {
```

Comment:
I don't see why would we ever need `ROVolume`. And even if we will need it we can add it later.

Reply:
Addressed [here](https://github.com/src-d/engine/pull/431/commits/4afbc44701165e9b523197e4cc8691c40428902d).